### PR TITLE
Add Zilliz Cloud Pipeline Text demo

### PR DESCRIPTION
**Pull request by jaelgu**

### CODE REVIEW

**Review by zc277584121 — COMMENTED** [review]
/lgtm
/approve

File: bootcamp/RAG/zilliz_pipeline_rag_text.ipynb [hunk 1]
```diff
@@ -0,0 +1,672 @@
+{
```
Comment (codingjaguar) [inline bootcamp/RAG/zilliz_pipeline_rag_text.ipynb:1]: <h1>Build RAG using Zilliz Cloud Pipelines for Text</h1><p> </p><p>标题体现一下和doc那篇的区别。另外我觉得这个最终我们想用一篇文章分别讲doc、text、image。现在先这样分开放着吧。</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (codingjaguar) [inline bootcamp/RAG/zilliz_pipeline_rag_text.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.                    "As the name suggests, Milvus Lite is a lightweight version that integrates seamlessly with Google Colab and Jupyter Notebook. It is packaged as a single binary with no additional dependencies, making it easy to install and run on your machine or embed in Python applications. Additionally, Milvus Lite includes a CLI-based Milvus standalone server, providing flexibility for running it directly on your machine. Whether you embed it within your Python code or utilize it as a standalone server is entirely up to your preference and specific application requirements.",
</pre><p> </p><p> </p><p> 用这几个语料我感觉不好，一个是讲milvus的话题读者还得反应一下，另外这几段描述都要马上变了，milvus的几个deployment modes定位会调整。想想其他语料，最好是一目了然的，不要让读者在理解语料上花功夫。比如用这个语料：<a href="https://milvus.io/docs/embeddings.md#Example-1-Use-default-embedding-function-to-generate-dense-vectors" target="_blank">https://milvus.io/docs/embeddings.md#Example-1-Use-default-embedding-function-to-generate-dense-vectors</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Pull request merged**